### PR TITLE
Fix for grid color changes not applying

### PR DESCRIPTION
**Pull request by misslivirose**
Fixes FB case 9376 - Edit grid always renders grey

### CODE REVIEW

**Comment by misslivirose** [comment]
**Test Plan**
1. In a domain where you have edit permissions, open up the Create menu and select the 'Grid' tab
2. Observe that the grid by default appears white (you may need to hit the 'Align to Avatar' button to see the grid depending on your location in the domain)
3. Change the color using the color picker and the individual RGB sliders
4. Confirm that color changes apply to the grid

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/8239/app_12701_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/8239/HighFidelity-Beta-PR12701-c66badc53c7b8e49a56dbc934cd07eeb528ea840.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/8239/HighFidelity-Beta-PR12701-c66badc53c7b8e49a56dbc934cd07eeb528ea840.exe

**Comment by conklin94122** [comment]
build this please

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/8275/app_12701_PR.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/8275/HighFidelity-Beta-PR12701-c66badc53c7b8e49a56dbc934cd07eeb528ea840.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/8275/HighFidelity-Beta-PR12701-c66badc53c7b8e49a56dbc934cd07eeb528ea840.dmg

File: scripts/system/libraries/gridTool.js [hunk 1]
```diff
@@ -1,16 +1,8 @@
 var GRID_CONTROLS_HTML_URL = Script.resolvePath('../html/gridControls.html');
 
-Grid = function(opts) {
+Grid = function (opts) {
```
Comment (thoys) [inline scripts/system/libraries/gridTool.js]: why the extra space after the function keyword?

File: scripts/system/libraries/gridTool.js [hunk 2]
```diff
@@ -28,58 +20,70 @@ Grid = function(opts) {
...
         minorGridEvery: minorGridEvery,
-        majorGridEvery: majorGridEvery,
+        majorGridEvery: majorGridEvery
     });
 
     that.visible = false;
     that.enabled = false;
 
-    that.getOrigin = function() {
+    that.getOrigin = function () {
```
Comment (thoys) [inline scripts/system/libraries/gridTool.js]: why the extra space after the function keyword?

File: scripts/system/libraries/gridTool.js [hunk 3]
```diff
@@ -28,58 +20,70 @@ Grid = function(opts) {
...
         return origin;
-    }
+    };
 
-    that.getMinorIncrement = function() { return minorGridEvery; };
-    that.setMinorIncrement = function(value) {
+    that.getMinorIncrement = function () {
```
Comment (thoys) [inline scripts/system/libraries/gridTool.js]: why the extra space after the function keyword?

File: scripts/system/libraries/gridTool.js [hunk 4]
```diff
@@ -28,58 +20,70 @@ Grid = function(opts) {
...
+        return minorGridEvery;
+    };
+
+    that.setMinorIncrement = function (value) {
```
Comment (thoys) [inline scripts/system/libraries/gridTool.js]: why the extra space after the function keyword?

File: scripts/system/libraries/gridTool.js [hunk 5]
```diff
@@ -28,58 +20,70 @@ Grid = function(opts) {
...
         minorGridEvery = value;
         updateGrid();
-    }
-    that.getMajorIncrement = function() { return majorGridEvery; };
-    that.setMajorIncrement = function(value) {
+    };
+
+    that.getMajorIncrement = function () {
```
Comment (thoys) [inline scripts/system/libraries/gridTool.js]: why the extra space after the function keyword?

File: scripts/system/libraries/gridTool.js [hunk 6]
```diff
@@ -28,58 +20,70 @@ Grid = function(opts) {
...
+        return majorGridEvery;
+    };
+
+    that.setMajorIncrement = function (value) {
```
Comment (thoys) [inline scripts/system/libraries/gridTool.js]: why the extra space after the function keyword?

File: scripts/system/libraries/gridTool.js [hunk 7]
```diff
@@ -28,58 +20,70 @@ Grid = function(opts) {
...
         majorGridEvery = value;
         updateGrid();
     };
 
-    that.getColorIndex = function() { return colorIndex; };
-    that.setColorIndex = function(value) {
-        colorIndex = value;
+    that.getColor = function () {
```
Comment (thoys) [inline scripts/system/libraries/gridTool.js]: why the extra space after the function keyword?

File: scripts/system/libraries/gridTool.js [hunk 8]
```diff
@@ -28,58 +20,70 @@ Grid = function(opts) {
...
+        return gridColor;
+    };
+
+    that.setColor = function (value) {
```
Comment (thoys) [inline scripts/system/libraries/gridTool.js]: why the extra space after the function keyword?

  + a few more below

File: scripts/system/libraries/gridTool.js [hunk 9]
```diff
@@ -28,58 +20,70 @@ Grid = function(opts) {
...
-    that.setEnabled = function(enabled) {
+    that.setEnabled = function (enabled) {
         that.enabled = enabled;
         updateGrid();
-    }
+    };
 
-    that.getVisible = function() { return that.visible; };
-    that.setVisible = function(visible, noUpdate) {
+    that.getVisible = function () { return that.visible; };
```
Comment (thoys) [inline scripts/system/libraries/gridTool.js]: return statement could be moved to next line for readability

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/8655/app_12701_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/8655/HighFidelity-Beta-PR12701-20ee1690547911038580ba48b166ea02f872a36b.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/8655/HighFidelity-Beta-PR12701-20ee1690547911038580ba48b166ea02f872a36b.exe

**Comment by qlimnavarro** [comment]
Currently Testing

**Comment by qlimnavarro** [comment]
Pass @conklin94122 
Color of grid can now be changed and is applied correctly. I did notice that after the interface restarts (Shutdown and then open up again) the color selcted for the grid does not save.

**Pull request merged**